roles/st2web/tasks/main.yml

```diff
@@ -0,0 +1,94 @@
+---
+# tasks file for st2web
+
+- name: Add nginx key
+  sudo: yes
+  apt_key:
+    url: http://nginx.org/keys/nginx_signing.key
+    state: present
+
+- name: Add nginx repos
+  sudo: yes
+  apt_repository:
+    repo: "deb http://nginx.org/packages/ubuntu/ trusty nginx"
+    state: present
+    update_cache: yes
+
+- name: Install nginx
+  sudo: yes
+  apt:
+    name: nginx=1.8.*
+    state: present
+
+- name: Install latest st2web package
+  sudo: yes
+  apt:
+    name: st2web
+    state: latest
+  when: st2_version == "stable"
+  notify:
+    - restart st2
+
+- name: Install exact st2web package
+  sudo: yes
+  apt:
+    name: st2web={{ st2_version }}-{{ st2web_revision }}
+    state: present
+  when: st2_version != "stable"
+  notify:
+    - restart st2
+
+- name: Create ssl cert dir
+  sudo: yes
+  file:
+    state: directory
+    dest: /etc/ssl/st2
+
+- name: Create ssl cert
+  sudo: yes
+  shell: openssl req -x509 -newkey rsa:2048 -keyout /etc/ssl/st2/st2.key -out /etc/ssl/st2/st2.crt -days XXX -nodes -subj "/C=US/ST=California/L=Palo Alto/O=StackStorm/OU=Information Technology/CN=$(hostname)"
```

Comment:
Instead of `-days XXX` (it's from our doc) we need something more explicit like `-days 365`.